snippets/capabilities-table.adoc

```diff
@@ -28,6 +28,9 @@ The following table describes the `baselineCapabilitySet` values.
 |`v4.16`
 |Specify this option when you want to enable the default capabilities for {product-title} 4.16. By specifying `v4.16`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.16 are `baremetal`, `MachineAPI`, `marketplace`, `OperatorLifecycleManager`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot`, `NodeTuning`, `ImageRegistry`, `Build`, `CloudCredential`, `DeploymentConfig`, and `CloudControllerManager`.
 
+|`v4.17`
```

Comment:
Maybe in a future PR consider adjusting the widths of these columns to give this column less room, to give more to the description column.